src/server.js

```diff
@@ -0,0 +1,69 @@
+import express from 'express';
+import path from 'path';
+import cookieParser from 'cookie-parser';
+import logger from 'morgan';
+import helmet from 'helmet';
+import Provider from 'oidc-provider';
+
+import { provider as providerConfiguration } from './config/provider';
+import { clients } from './config/clients';
+import { mountRoutes } from './router';
+
+const { PORT = 5000, ISSUER = `http://localhost:${PORT}` } = process.env;
+
+const server = express();
+
+// View engine setup
+server.set('views', path.join(__dirname, 'views'));
+server.set('view engine', 'ejs');
+server.use((req, res, next) => {
+  // cheap layout implementation for ejs
+  const orig = res.render;
+  res.render = (view, locals) => {
+    server.render(view, locals, (err, html) => {
+      if (err) throw err;
+      orig.call(res, '_layout', {
+        ...locals,
+        body: html,
+      });
+    });
+  };
+  next();
+});
+
+// Other middlewares setup
+server.use(helmet());
+server.use(logger('dev'));
+server.use(cookieParser());
+server.use('/assets', express.static(path.join(__dirname, 'public')));
+
+const provider = new Provider(ISSUER, providerConfiguration);
+
+server.provider = provider;
+
+server.start = function start() {
```

Comment:
no need to name the function here, we can directly use : 

```
server.start = () => {
```

Reply:
This was introduced to please the linter ([func-name rule](https://eslint.org/docs/rules/func-names)).
Should we disable this rule in eslint configuration?

Reply:
my bad, I prefer to stick to airbnb configuration.